db/warehouse/migrate/20200625130802_improve_db_performance.rb

```diff
@@ -0,0 +1,63 @@
+class ImproveDbPerformance < ActiveRecord::Migration[5.2]
+  # Concurrent indexes can't run in a transaction
+  disable_ddl_transaction!
+
+  def change
+    GrdaWarehouse::ServiceHistoryService.sub_tables.each do |year, table|
+
+      begin
+        add_index table, [:client_id], name: "index_shs_#{year}_client_id_only", algorithm: :concurrently
+      rescue ArgumentError
+        puts "Skipping index_shs_#{year}_client_id_only which already exists"
+      end
+
+      begin
+        add_index table, [:service_history_enrollment_id], name: "index_shs_#{year}_en_id_only", algorithm: :concurrently
+      rescue ArgumentError
+        puts "Skipping index_shs_#{year}_en_id_only which already exists"
+      end
+
+      conditionally_add_statistics_if_supported(year, table)
+    end
+
+    add_index :warehouse_clients_processed, :client_id, algorithm: :concurrently
+    add_index :warehouse_clients, :data_source_id, algorithm: :concurrently
+    add_index :ProjectCoC, :data_source_id, algorithm: :concurrently
```

Comment:
Do these need any sort of rescue?  Presumably after this runs they'll be included in the schema file that gets loaded for new deploys in db_prep, so they would exist for new installations.

Reply:
I believe these would make it to new deployments through this migration until we create a new structure.sql, but they should be in the schema.rb file. That said, it looks like running db/prep can fail without github actions noticing. I'll fix that first.

They do need rescues since it's not transnational and could maybe have run already on a previous try. I'll fix that too.